kotlin-lotto/src/main/kotlin/lotto/model/Lotto.kt

```diff
@@ -0,0 +1,55 @@
+package lotto.model
+
+import camp.nextstep.edu.missionutils.Randoms
+import lotto.validation.requireLotto
+
+const val LOTTO_SIZE = 6
+const val LOTTO_MIN_INCLUDE = 1
+const val LOTTO_MAX_INCLUDE = 45
+const val LOTTO_PRICE = 1000
+
+class Lottos(val lottos: List<Lotto>) {
+    val lottoCount = lottos.size
+    val purchasePrice = lottoCount * LOTTO_PRICE
+
+    companion object {
+        fun generate(purchasePrice: PurchasePrice): Lottos {
+            val lottoCount = purchasePrice.price / LOTTO_PRICE
+            return Lottos((1..lottoCount).map { Lotto.generate() })
```

Comment:
```suggestion
            return List(lottoCount) { Lotto.generate() }
```

이번 미션할 때 처음 알았는데 이렇게도 되더라고요! 선배님 생각은 어떠신지 궁금합니다~

Reply:
오 이게 훨씬 좋네요 👍👍